zgw/zaken-api/src/main/resources/config/liquibase/changelog/20240214-create-table-zaak-hersteltermijn.xml

```diff
@@ -37,6 +37,13 @@
         </createTable>
     </changeSet>
 
+    <changeSet id="2" author="Ritense">
+        <createIndex indexName="idx_zaak_hersteltermijn_zaak_url_end_date" tableName="zaak_hersteltermijn" unique="true">
+            <column name="zaak_url"/>
+            <column name="end_date"/>
+        </createIndex>
```

Comment:
Your version would not allow 2 hersteltermijnen to be ended on the same day. The following does, because the index is only applied to rows where end_date is null:
```suggestion
         <preConditions onFail="MARK_RAN">
            <dbms type="postgresql"/>
        </preConditions>
        <sql>
            CREATE UNIQUE INDEX idx_zaak_hersteltermijn_zaak_url_end_date ON zaak_hersteltermijn (zaak_url, end_date) WHERE end_date IS NULL;
        </sql>
```

Reply:
Changed